data/dataset.urls

```diff
@@ -0,0 +1,45 @@
+# Dataset urls required fo tutorial, described at https://esmvalgroup.github.io/ESMValTool_Tutorial/setup.html#using-your-own-machine
+# For Running a recipe episode
+# Variable = thetaoga, model = HadGEM2-ES
```

Comment:
For the first exercise in the preprocessor episode we also need the variable "thetao" which is much larger (a several GB file for every 10 years) than "thetaoga".
These are not here yet..

Reply:
But maybe it should be reconsidered whether we need all these different datasets. We could just stick with one or two different ones maybe, something for the next esmvaltool tutorial sprint?

Reply:
Ah I missed thetao because its not present in the solution recipe

Reply:
Yeah, maybe that's an issue in itself?

Reply:
In #110 once we test solution recipes we would also have caught it.